pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/realtime/segment/SegmentSizeBasedFlushThresholdUpdater.java

```diff
@@ -50,5 +57,9 @@ public synchronized void updateFlushThreshold(StreamConfig streamConfig, Segment
         _flushThresholdComputer.computeThreshold(streamConfig, committingSegmentDescriptor, committingSegmentZKMetadata,
             newSegmentZKMetadata.getSegmentName());
     newSegmentZKMetadata.setSizeThresholdToFlushSegment(threshold);
+
+    _controllerMetrics.setOrUpdateTableGauge(_rawTableName, ControllerGauge.NUM_ROWS_THRESHOLD, threshold);
```

Comment:
Do we want to log it under the raw name or real-time table name (with suffix)? Seems most of the gauges are under the table name with suffix

Reply:
I thought about it and decided on raw name. But I don't have a preference one way or another.